src/carts/src/carts-service/services.py

```diff
@@ -0,0 +1,180 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+import json
+from datetime import datetime, timedelta
+from uuid import uuid4
+
+import boto3
+from flask import request
+from server import app
+from dynamo_setup import dynamo_client, ddb_table_carts
+from botocore.exceptions import ClientError
+
+from werkzeug.exceptions import BadRequest
+
+
+class CartService:
+    dynamo_client = dynamo_client
+    ddb_table_carts = ddb_table_carts
+    cart_ttl_days = 7
+
+    @staticmethod
+    def marshal_item(data):
```

Comment:
Just out of curiosity, why do we marshal/unmarshal the cart this way rather than using the json library?

Reply:
I didn't know dynamodb items could be marshalled/unmarshalled using the json library. I'd prefer a standard way if there is one, rather than piecing together my own method. Could you point me to a resource or anything that shows how to do that?

Reply:
My bad, not the json library for DynamoDB but boto3 has the TypeSerializer/Deserializer https://boto3.amazonaws.com/v1/documentation/api/latest/_modules/boto3/dynamodb/types.html